index.html

```diff
@@ -10,7 +10,7 @@
   <body>
     <main id="main">
       <h1 id="title">Dr. Norman Borlaug</h1>
-      <p>The man who saved a billion lives</p>
+      <p>The man who saved a billion of live</p>
```

Comment:
you made an extraordinary effort to make this project happen and I think you can make it even decent if you replace the <p> tag with the h1 title because this is the main idea or point on the page that needs a lot of attention as this makes it more semantically powerful you can easily make it like this  what do think<h1>The man who saved a billion lives</h1>